msg/OrientationConstraint.msg

```diff
@@ -8,10 +8,20 @@ geometry_msgs/Quaternion orientation
 # The robot link this constraint refers to
 string link_name
 
-# optional axis-angle error tolerances specified
+# Optional symmetric error tolerances specified
 float64 absolute_x_axis_tolerance
 float64 absolute_y_axis_tolerance
 float64 absolute_z_axis_tolerance
 
+# How should we parameterize the optional orientation error tolerance?
+# See absolute axis tolerance below to specify the manitude.
+uint8 parameterization
+
+# The different options for the orientation error parameterization
+# Intrinsic xyz Euler angles (default value)
+uint8 XYZ_EULER_ANGLES=0
+# A rotation vector, the norm represents the rotation angle around this vector
```

Comment:
```suggestion
# (Axis-angle representation) A rotation vector. The norm represents the rotation angle around this vector
```
Come to think of it, we could just call this setting `AXIS_ANGLE`, since axis-angle seems to be a fixed term: https://en.wikipedia.org/wiki/Axis%E2%80%93angle_representation

Reply:
I think angle-axis refers to using the angle and the axis separately. As mentioned on the wiki page you linked:
> The axis–angle representation is equivalent to the more concise rotation vector, also called the Euler vector

But I'm certainly not calling it an Euler vector :)

However, I'm open to changing it back to "angle-axis". Maybe it is easier to interpret than "rotation vector".

Reply:
I'm good with calling it `ROTATION_VECTOR` if we reference the name "axis-angle" as well.

Reply:
Maybe
```
# A rotation vector, this is similar to the angle-axis representation
# but the rotation axis is multiplied with the rotation angle.
```

Btw, thanks for reviewing the documentation @felixvd! It is really difficult to judge my own comments because I've to much background on the details :)

Reply:
I think this would be great (note the punctuation change, similar to what Felix suggested):

```
# A rotation vector. This is similar to the angle-axis representation
# but the rotation axis is multiplied with the rotation angle.
uint8 ROTATION_VECTOR=1
```

Reply:
"the axis is multiplied with the angle" would confuse me, I think. "The magnitude of the vector represents the rotation angle" (almost like before) seems good to me.